SlicerCART/src/SlicerCART.py

```diff
@@ -1546,11 +1619,12 @@ def cast_segmentation_to_uint8(self):
           input_path = os.path.basename(case)
           if input_path.endswith('.nii') or input_path.endswith('.nii.gz'):
               segm = nib.load(case)
-              segm_data = segm.get_fdata()
+              segm_data_dtype = segm.dataobj.dtype
               print(f'infile: {input_path}, dtype: {segm_data.dtype}')
-              if segm_data.dtype != np.uint8:
+              if segm_data_dtype != np.uint8:
                   segm_data = segm_data.astype(np.uint8)
-                  segm_nii = nib.Nifti1Image(segm_data, segm.affine)
+                  segm.header.set_data_dtype(np.uint8)
+                  segm_nii = nib.Nifti1Image(segm_data, segm.affine, segm.header)
```

Comment:
These changes seem unrelated to package management. For the sake of clarity in the git history, should we do avoid unrelated changes in this PR, and address these changes in another, more specific, PR?

Reply:
Yes, sorry, I should have created another issue and PR specific to this, a small bug I found while testing the module. Note: `.get_fdata()` returns a floating point image so `if segm_data.dtype != np.uint8:` would always have returned `False`.